lib/spack/spack/util/environment.py

```diff
@@ -25,23 +25,10 @@
 import os
 
 system_paths = ['/', '/usr', '/usr/local']
-suffixes = ['lib', 'lib64', 'include']
-system_dirs = [os.path.join(p, s) for s in suffixes for p in system_paths] + \
-    system_paths
-system_bins = [os.path.join(p, 'bin') for p in system_paths]
 
 
 def filter_system_paths(paths):
-    return [p for p in paths if p not in system_dirs]
```

Comment:
I can see why the extra (unused) functionality would be useful. I guess I can add it back. I think I'll also add filtering of bin while I'm at it. Any other directories you think we should filter?